packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -111,5 +113,10 @@ export const messages = {
     'A refresh of your sObject definitions is already underway. If you need to restart the process, cancel the running task.',
   force_project_create_text: 'SFDX: Create Project',
   force_project_create_open_dialog_create_label: 'Create Project',
-  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger'
+  force_apex_trigger_create_text: 'SFDX: Create Apex Trigger',
+
+  isv_debug_bootstrap_step1_create_project:
+    'SFDX: ISV Debugger Setup Step 1: Create Project',
+  isv_debug_bootstrap_step1_configure_project:
+    'SFDX: ISV Debugger Setup Step 2: Configure Project'
```

Comment:
@ruthemmanuelle new text to be shown when showing progress in the VS Code output

Reply:
How about these?

isv_debug_bootstrap_step1_create_project:
  'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
isv_debug_bootstrap_step1_configure_project:
  'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project'

Reply:
Done.